# DCAP
DCAP:Integrating multi-omics data with deep learning for predicting cancer prognosis
DCAP is designed to integrate multi-omics data for predicting the prognosis risk of cancer patients.The multi-omics data was input into Autoencoder to obtain representative composite features that can best rebuild all input data. These features were then input to the Cox proportional hazards (Cox-PH) model to estimate the patients’ prognosis risks. Finally, in order to reduce the number of features for cancer prognosis prediction, we employed XGboost for features selection, and reconstruct the cancer prognosis model.

# Requirements
tensorflow, python 3, R

# Data preparation
In this study, we utilized cancer datasets from the TCGA portal (https://tcga-data.nci.nih.gov/tcga/). All these datasets were downloaded by using the R package “TCGA-assembler”(v1.0.3, (Wei, et al., 2018)), which contains four types of multi-omics data: mRNA, miRNA, DNA methylation, and copy number variation (CNV) data. Here, “mRNA” was RNA sequencing data generated by UNC Illumina HiSeq_RNASeq V2; Level 3, “miRNA” was miRNA sequencing data obtained by BCGSC Illumina HiSeq miRNASeq, DNA methylation data was generated by USC HumanMethylation450, and CNV data that generated by BROAD-MIT Genome wide SNP_6. 

# Steps
Firstly please use the 
